app/controllers/copy_projects_controller.rb

```diff
@@ -75,10 +75,9 @@ def copy_project
   private
 
   def prepare_for_copy_project
-    @issue_custom_fields = WorkPackageCustomField.find(:all, order: "#{CustomField.table_name}.position")
+    @issue_custom_fields = WorkPackageCustomField.order("#{CustomField.table_name}.position")
     @types = ::Type.all
-    @root_projects = Project.find(:all,
-                                  conditions: "parent_id IS NULL AND status = #{Project::STATUS_ACTIVE}",
-                                  order: 'name')
+    @root_projects = Project.where("parent_id IS NULL AND status = #{Project::STATUS_ACTIVE}")
```

Comment:
<pre>
where(parent_id: nil, status: Project::STATUS_ACTIVE)
</pre>

would be better still.